rnd/autogpt_server/test/util/test_service.py

```diff
@@ -29,7 +30,7 @@ async def add_async(a: int, b: int) -> int:
 @pytest.mark.asyncio(scope="session")
 async def test_service_creation(server):
     with TestService():
-        client = get_service_client(TestService)
+        client = get_service_client(TestService, 8005)
```

Comment:
These hardcoded ports can be the from the config value? since the default has already been defined.

Reply:
i updated it where I'd forgotten, just when getting scheduler, it doesn't work with 8004 in the tests as that's for the agent server daemon, if that's what you mean here?